apcd-cms/src/apps/extension/views.py

```diff
@@ -8,46 +8,55 @@
 
 logger = logging.getLogger(__name__)
 
-class ExtensionFormView(TemplateView):
-    template_name = 'extension_submission_form/extension_submission_form.html'
+
+class ExtensionFormView(TemplateView):      
```

Comment:
```suggestion
class ExtensionFormView(TemplateView):
```
Removing excess space